### DISCUSSION TITLE
erro ao rodar

### DISCUSSION THREAD

**Discussion by hedesjr** [rtenorioh/Press-Ticket#90] [General]
Olá, tudo bem?
Estou fazendo uma instalação do seu material em um servidor aapanel conforme o video do canal do conhecimento https://www.youtube.com/watch?v=fWUOne25jfo segui os passos direitinho que ele mostra no video seguindo seu material, mas ao rodar o app ele da este erro: Error: 

ENOENT: no such file or directory, stat '/www/wwwroot/ticket/frontend/build/index.html'

ele esta neste link https://app.fatoo.com.br/ 
Gostaria que pudesse me auxiliar por favor, obrigado.

**Comment by rtenorioh** [comment]
espero que tenha resolvido seu problema, o video trata de uma versão diferente ao do repositorio, pois precisa criar manualmente um arquivo no front, por nao criar o config.json, não consegue realizar a bild do front, automaticamente ficando sem o index.html, no manual de instalação em VPS tem o comando

**Comment by rtenorioh** [comment]
como meio de evitar esses problemas vou trabalhar agora para termos no front um local para setar essas informações, ficando mais facil personalizar o Press Ticket e deixando-o mais profissional. peço desculpas pela demora em responder.

**Comment by rtenorioh** [comment]
Erro corrigido que será liberado na v1.10.0, onde exclui o arquivo config.json e criei uma area onde pode inserir as cores, logos e dados da empresa e salvando no BD, assim evitando erros durante a instalação e melhorando a personalização

![image](https://github.com/user-attachments/assets/04f5c437-69ed-4de8-a4d2-4217bc389389)

Caso queira testar a nova versão segue acesso:

**Versão BETA**

[https://beta.pressticket.com.br/](https://beta.pressticket.com.br/)

Usuário padrão para Acesso 

Usuário: [admin@pressticket.com.br](mailto:admin@pressticket.com.br) 
Senha: admin 

Usuário Master para Acesso 

Usuário: [masteradmin@pressticket.com.br](mailto:masteradmin@pressticket.com.br) 
Senha: masteradmin
